src/components/navbar/NavBar.jsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames'
+import React, { useContext } from 'react'
+import { Link } from 'react-router-dom';
+import { Context } from '../..'
+import cl from './NavBar.module.css'
+import MyLogo from '../logo/MyLogo';
+import { ADMIN_ROUTE, BASKET_ROUTE, CONTACTS_ROUTE, HOME_ROUTE, LOGIN_ROUTE, REGISTRATION_ROUTE, SHOP_ROUTE, USER_ROUTE } from '../../utils/consts';
```

Comment:
```suggestion
import {
ADMIN_ROUTE,
BASKET_ROUTE,
CONTACTS_ROUTE,
HOME_ROUTE,
LOGIN_ROUTE,
REGISTRATION_ROUTE,
SHOP_ROUTE,
USER_ROUTE
} from '../../utils/consts';
```
line should not be longer then 200 chars